assets/src/edit-story/components/checklist/accessibilityChecks.js

```diff
@@ -13,6 +13,25 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
+ * Internal dependencies
+ */
+import ElementLinkTappableRegionTooSmall from './checks/elementLinkTappableRegionTooSmall';
+import ImageElementMissingAlt from './checks/imageElementMissingAlt';
+import { PageBackgroundTextLowContrast } from './checks/pageBackgroundLowTextContrast';
```

Comment:
Out of curiosity, @embarks or @littlemilkstudio why was this one put in its own directory?